### ISSUE TITLE
Open Source Mycetoma (MycetOS) Monthly Meeting 27th July 2021

### ISSUE THREAD

**Issue by fantasy121** [OpenSourceMycetoma/Series-1-Fenarimols#62]
### **Open Source Mycetoma (MycetOS) Meeting 27th July 2021**  

**Time:** UTC 11:00 - London Noon - Amsterdam 1 PM - Sydney 9 PM - New York 7 AM

**Chair:** USYD @fantasy121  

**Minute taker:** Erasmus University Team @wwjvdsande to appoint minute taker

_Recording_ can be found here  

_Minutes_ can found here  

**Link to previous meeting** #58  
____________________________________________
### 1. Outstanding items from previous meeting 

> - [ ] a) ISNTD: Mat will check with organisers of the conference to confirm about presentation of MycetOS research at this conference this year. @mattodd 
> - [ ] b) Dmitrij to ship his next batch of synthesised compounds @dmitrij176 
> - [ ] c) Hung to complete SAR of Fenarimol analogues @fantasy121 
> - [ ] d) A solution to distinguish retested compounds in Master List. (look at OSM master list as an example)
> - [ ] e) Chemical synthesis and characterisation to be added to manuscript #55  @kym834 @fantasy121 @dmitrij176 

### 2. Discussion points for this meeting  

- Next stage of manuscript

### 3. Any other businesses?

Meeting adjourned.

### 4. Action Items for Aug 2021 Meeting

[Template for monthly meeting GitHub Issue Posts can be found in #59 ]

**Comment by fantasy121** [comment]
@OpenSourceMycetoma/corecontrib Meeting details for July meeting is now up. Apology for the late notice.

**Comment by wwjvdsande** [comment]
Action items based on this meeting:
1.	Matt will inform if there is an update on the ISNTD conference 
2.	Ma will perform in vitro susceptibility testing on the compounds and will inform everyone on github on the outcomes
3.	The SAR performed by Hung and Kymberly will be checked by Matt and Dimitrij
4.	Synthesis of compounds and SAR will be added to manuscript
5.	The compounds needed for in vivo testing based on the SAR will be requested from epichem
6.	We should streamline the requests from schools to make series 2 compounds. So far it looks like an ideal project in which schools can participate
7.	Series 5 seems so far an ideal series to look further into. Wendy will ask MMV for more details on the compound series with a hit to check if there are more analogues which can be tested in vitro.

**Comment by fantasy121** [comment]
@MA-Jjingyi @mkonings Martine just emailed me and told me that the requested Epichem compounds have been re-shipped. Please look out for them. Thanks.

**Comment by MA-Jjingyi** [comment]
@fantasy121 Yep. I will pay attention to it. I will let you know when I receive them.

**Comment by MA-Jjingyi** [comment]
@fantasy121 we have received the Epichem compounds from Martine. thanks. we plan to do vivo test recently.

**Comment by fantasy121** [comment]
@OpenSourceMycetoma/corecontrib re: August meeting. I want to suggest we don’t have a meeting in August and resume meeting in September instead. This is because Wendy and Mat are on leave, according to July meeting, and Sydney is in lockdown so there won’t be significant updates from USYD. What does everyone think?

Are Mat and Wendy still going to be away for this meeting? I’m still happy to join the meeting if everyone wants to keep this month’s meeting.

**Comment by dmitrij176** [comment]
> @OpenSourceMycetoma/corecontrib re: August meeting. I want to suggest we don’t have a meeting in August and resume meeting in September instead. This is because Wendy and Mat are on leave, according to July meeting, and Sydney is in lockdown so there won’t be significant updates from USYD. What does everyone think?
> 
> Are Mat and Wendy still going to be away for this meeting? I’m still happy to join the meeting if everyone wants to keep this month’s meeting.

@fantasy121 I was on holiday and dont have any updates either. Unless there are objections, I would postpone our meeting to next month.
